internal/attestation/measurements/measurements_test.go

```diff
@@ -1043,3 +1045,8 @@ func TestMeasurementsCompare(t *testing.T) {
 		})
 	}
 }
+
+func TestRekorErrCheck(t *testing.T) {
+	err := fmt.Errorf("%w: %w", ErrRekor, errors.New("test"))
+	assert.ErrorIs(t, err, ErrRekor)
+}
```

Comment:
```suggestion
```
Should be in `internal/attestation/measurements/fetchmeasurements_test.go`, if at all